polaris-react/src/components/Tile/Tile.tsx

```diff
@@ -3,38 +3,44 @@ import type {spacing} from '@shopify/polaris-tokens';
 
 import styles from './Tile.scss';
 
+type Breakpoints = 'xs' | 'sm' | 'md' | 'lg' | 'xl';
+
 type SpacingTokenName = keyof typeof spacing;
 
 // TODO: Bring this logic into tokens
-type Spacing = SpacingTokenName extends `space-${infer Scale}` ? Scale : never;
-
-type Columns =
-  | '1'
-  | '2'
-  | '3'
-  | '4'
-  | '5'
-  | '6'
-  | '7'
-  | '8'
-  | '9'
-  | '10'
-  | '11'
-  | '12';
+type SpacingScale = SpacingTokenName extends `space-${infer Scale}`
+  ? Scale
+  : never;
+
+type Columns = {
+  [Breakpoint in Breakpoints]?: number | string;
+};
+
+type Gap = {
+  [Breakpoint in Breakpoints]?: SpacingScale;
+};
 
 export interface TileProps {
   /** Elements to display inside tile */
-  children: React.ReactNode;
+  children?: React.ReactNode;
```

Comment:
Is there more context for why children is now an optional prop?

Reply:
Changing back!